R/pkg/R/functions.R

```diff
@@ -3652,3 +3652,43 @@ setMethod("posexplode",
             jc <- callJStatic("org.apache.spark.sql.functions", "posexplode", x@jc)
             column(jc)
           })
+
+#' collect_list
+#'
+#' Aggregate function: returns a list of objects with duplicates.
```

Comment:
does other function has `Aggregate function: ` in the description or this is carried over from Scala doc?
if latter we could go without - there's already a `@family` tag